docs/getting-started-guides/docker.md

```diff
@@ -55,35 +55,7 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
-2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
I agree . This shouldn't be removed completely, but we can move it to the troubleshooting section.